#
# http://patchwork.ozlabs.org/patch/62931/
# had to be re-written because it did not apply cleanly
#
--- gcc-4.5.2/gcc/testsuite/gcc.dg/vect/pr43430-1.c	2010-04-04 08:30:54.000000000 -0700
+++ gcc-4.5.2/gcc/testsuite/gcc.dg/vect/pr43430-1.c	2013-02-12 15:37:22.692052970 -0800
@@ -35,5 +35,5 @@
   return foo (data_ch1, data_ch2, 1);
 }
 
-/* { dg-final { scan-tree-dump-times "vectorized 1 loops" 1 "vect" } } */
+/* { dg-final { scan-tree-dump-times "vectorized 1 loops" 1 "vect" { target vect_condition } } } */
 /* { dg-final { cleanup-tree-dump "vect" } } */
